<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Group_Dashboard</class>
 <widget class="QMainWindow" name="Group_Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>671</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>ClockWork</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="logo">
      <property name="minimumSize">
       <size>
        <width>149</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="text">
       <string>Logo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>594</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="log_out_btn">
      <property name="text">
       <string>Log-Out</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="profile_button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10000000</width>
        <height>10000000</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Group Members</string>
      </property>
      <property name="text">
       <string>Profile</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="btns_frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="dashboard_btn">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Dashboard</string>
         </property>
         <property name="text">
          <string>Dashboard</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="activity_log_btn">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Activity Log</string>
         </property>
         <property name="text">
          <string>Activity Log</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="groups_btn">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Group Members</string>
         </property>
         <property name="text">
          <string>Groups</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="calendar_btn">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Task Calendar</string>
         </property>
         <property name="text">
          <string>Calendar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QStackedWidget" name="stacked_Manager">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="dashboard">
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="2" column="1">
         <widget class="QGroupBox" name="currenttask_box">
          <property name="title">
           <string>Current Tasks</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QTableWidget" name="currenttask_table">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>180</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>50</number>
             </attribute>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>Task</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Members/Groups Assigned</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Priority Level</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Due Date</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QGroupBox" name="task_calendar_box">
          <property name="title">
           <string>Task Calendar</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QCalendarWidget" name="Task_Calendar_2">
             <property name="toolTip">
              <string>Double-click the day you want to access tasks</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="task_completion_groupbox">
          <property name="title">
           <string>Task Completion</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="pie_chart">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>250</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::Box</enum>
             </property>
             <property name="text">
              <string>Pie Chart</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QGroupBox" name="created_groups_groupbox">
          <property name="title">
           <string>Number of Created Groups:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="number_groups_label">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::Box</enum>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="activity_log">
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <widget class="QGroupBox" name="activity_log_box">
          <property name="title">
           <string>Activity Log</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="3" column="0" colspan="2">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>387</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="assigntask_btn">
             <property name="text">
              <string>Assign Task</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="removetask_btn">
             <property name="text">
              <string>Remove Task</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QPushButton" name="validatetask_btn">
             <property name="text">
              <string>Validate Task</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="4">
            <widget class="QLineEdit" name="search_bar_tasks"/>
           </item>
           <item row="2" column="0" colspan="5">
            <widget class="QTabWidget" name="task_tab">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>Completed Tasks</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="0">
                <widget class="QTableWidget" name="completedtasks_table">
                 <property name="horizontalScrollMode">
                  <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
                 </property>
                 <attribute name="horizontalHeaderCascadingSectionResizes">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>200</number>
                 </attribute>
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderDefaultSectionSize">
                  <number>50</number>
                 </attribute>
                 <row/>
                 <row/>
                 <row/>
                 <row/>
                 <row/>
                 <column>
                  <property name="text">
                   <string>Tasks</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Members/Groups Assigned</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Status</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Priority Level</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Due Date</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_2">
              <attribute name="title">
               <string>Pending Tasks</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="0" column="0">
                <widget class="QTableWidget" name="pendingtasks_table">
                 <property name="horizontalScrollMode">
                  <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
                 </property>
                 <attribute name="horizontalHeaderCascadingSectionResizes">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>200</number>
                 </attribute>
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderDefaultSectionSize">
                  <number>50</number>
                 </attribute>
                 <row/>
                 <row/>
                 <row/>
                 <row/>
                 <row/>
                 <column>
                  <property name="text">
                   <string>Tasks</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Members/Groups Assigned</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Status</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Priority Level</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Due Date</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QPushButton" name="search_btn">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="4">
            <widget class="QLabel" name="search_tasks_label">
             <property name="text">
              <string>Search Tasks</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="task_calendar">
       <layout class="QGridLayout" name="gridLayout_17">
        <item row="0" column="0">
         <widget class="QCalendarWidget" name="Task_Calendar">
          <property name="toolTip">
           <string>Double-click the day you want to access tasks</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="reminder_groupbox">
          <property name="title">
           <string>Reminders</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QFrame" name="tab_reminder">
             <layout class="QGridLayout" name="gridLayout_16">
              <item row="0" column="0" colspan="2">
               <widget class="QTableWidget" name="reminders_table">
                <attribute name="horizontalHeaderCascadingSectionResizes">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderMinimumSectionSize">
                 <number>150</number>
                </attribute>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>150</number>
                </attribute>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderMinimumSectionSize">
                 <number>50</number>
                </attribute>
                <attribute name="verticalHeaderDefaultSectionSize">
                 <number>50</number>
                </attribute>
                <row/>
                <row/>
                <row/>
                <column>
                 <property name="text">
                  <string>Reminder</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Date</string>
                 </property>
                </column>
               </widget>
              </item>
              <item row="1" column="0">
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>195</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="add_reminder_button">
                <property name="text">
                 <string>Add Reminder</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="group_window">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="delete_group_button">
          <property name="text">
           <string>Delete Group</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="create_group_button">
          <property name="text">
           <string>Create Group</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="check_group_members_button">
          <property name="text">
           <string>Check Group Members</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="5">
         <widget class="QGroupBox" name="box_of_groups">
          <property name="title">
           <string>Created Groups</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLineEdit" name="search_bar_groups"/>
           </item>
           <item row="2" column="0">
            <widget class="QTableWidget" name="groups_table">
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
             </property>
             <property name="rowCount">
              <number>5</number>
             </property>
             <property name="columnCount">
              <number>3</number>
             </property>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>160</number>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>160</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>50</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>50</number>
             </attribute>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>Group Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Members</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Submission Status</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="search_group_label">
             <property name="text">
              <string>Search Group</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
